File: test/aidbox_sdk/snapshots/dotnet/hl7-fhir-r4-core/MetadataResource.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.R4.Core;

public class MetadataResource : DomainResource, IResource
{
    public string? Description { get; set; }
    public string? Date { get; set; }
    public string? Publisher { get; set; }
    public Base.CodeableConcept[]? Jurisdiction { get; set; }
    public string? Name { get; set; }
    public Base.UsageContext[]? UseContext { get; set; }
    public bool? Experimental { get; set; }
    public string? Title { get; set; }
    public required string Status { get; set; }
    public string? Url { get; set; }
    public string? Version { get; set; }
    public Base.ContactDetail[]? Contact { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/MetadataResource.py
from typing import Optional, List
from base import UsageContext
from base import ContactDetail
from base import CodeableConcept
from base import DomainResource

class MetadataResource(DomainResource):
    description: Optional[str] = None
    date: Optional[str] = None
    publisher: Optional[str] = None
    jurisdiction: Optional[List[CodeableConcept]] = None
    name: Optional[str] = None
    use_context: Optional[List[UsageContext]] = None
    experimental: Optional[bool] = None
    title: Optional[str] = None
    status: str
    url: Optional[str] = None
    version: Optional[str] = None
    contact: Optional[List[ContactDetail]] = None